lib/models/device.coffee

```diff
@@ -49,6 +49,12 @@ deviceStatus = require('balena-device-status')
 	getDeviceOsSemverWithVariant
 	normalizeDeviceOsVersion
 } = require('../util/device-os-version')
+{
+	checkLocalModeSupported
+	getLocalModeSupport
+	LOCAL_MODE_ENV_VAR
+	LOCAL_MODE_SUPPORT_PROPERTIES
+} = require('../util/local-mode')
```

Comment:
Chose to implement this in TS and just re-export the methods.

Reply:
I think that's the right approach and we should try to do that whenever possible.

Reply:
I agree @sradevski . Atm there a big `util/index` which also re-exports all other utils and the models use that, which makes me sad :(
I indeed want to split lots of them and use explicit imports for (almost) all of them, which
* can make the CLI load faster, since it will be able to avoid some dependencies of the utils
* possibly allow us to use `util/index` for exporting some functions to the SDK consumers.